### ISSUE TITLE
Add support for \begin{rcases}

### ISSUE THREAD

**Issue by themattgabriel** [KaTeX/KaTeX#2144]
**Describe the solution you'd like:**

```
$$
\text {description} \begin{rcases}
\text{item 1} \\
\text{item 2}
\end{rcases}
$$
```
The following lines above would render a **right brace** instead and swap the positions of *description* and *items*.
![Annotation 2019-11-07 211534](https://user-images.githubusercontent.com/51425211/68449586-f5e65a00-01a4-11ea-9b9c-5812bc2bd263.png)

**Link to or name of a (La)TeX package that provides the same feature:**
According to [this reply](https://tex.stackexchange.com/questions/47560/how-to-put-a-brace-on-the-right-not-left-to-group-cases), this can be done using (La)TeX package `mathtools`.

**Comment by ronkok** [comment]
@themattgabriel Thank you for the suggestion. This is pretty easy to implement. I'll work up a PR.

`{rcases}`, by the way, does not "swap the positions of *description* and *items*". The document author has to do that. You can see an example on page 19 or the `mathtools` [instructions](http://www.texdoc.net/texmf-dist/doc/latex/mathtools/mathtools.pdf).

**Comment by themattgabriel** [comment]
@ronkok Thank you.

> {rcases}, by the way, does not "swap the positions of description and items". The document author has to do that.

You're right. I meant that to be my intention rather than the output.

**Comment by ylemkimon** [comment]
Fixed via #2149.
